src/app/src/sensorUtils.js

```diff
@@ -112,3 +118,50 @@ export function transformSensorDataToRatings(sensorData) {
         },
     };
 }
+
+export function getFishIconForVariableRating(rating) {
+    // TODO: Figure out how the warning state is defined
```

Comment:
Consider creating an issue and referencing it here (for example TODO #999: Figure out ...). Just to make sure we get back to the TODOs.